docs/usage/shoot_status.md

```diff
@@ -99,6 +99,7 @@ Known error codes are:
 - `ERR_INFRA_INSUFFICIENT_PRIVILEGES` - indicates that the last error occurred due to insufficient infrastructure privileges. It is classified as a non-retryable error code.
 - `ERR_INFRA_QUOTA_EXCEEDED` - indicates that the last error occurred due to infrastructure quota limits. It is classified as a non-retryable error code.
 - `ERR_INFRA_DEPENDENCIES` - indicates that the last error occurred due to dependent objects on the infrastructure level. It is classified as a non-retryable error code.
+- `ERR_RETRYABLE_INFRA_DEPENDENCIES` - indicates that the last error occurred due to dependent objects on  on the infrastructure level. "Retryable" means that the occurred error is likely to be resolved in a ungraceful manner after given period of time.
```

Comment:
This statement about "retryable" with `is likely to be resolved in a ungraceful manner after given period of time` is applicable for `ERR_RETRYABLE_CONFIGURATION_PROBLEM`. I guess there are no `ungraceful manners` in the `ERR_RETRYABLE_INFRA_DEPENDENCIES`, so le't reword this to avoid confusion.

```suggestion
- `ERR_RETRYABLE_INFRA_DEPENDENCIES` - indicates that the last error occurred due to dependent objects on the infrastructure level, but the operation should be retried.
```